RELEASE_NOTES.md

```diff
@@ -1,5 +1,521 @@
+
+See <https://medium.com/rubyinside/whats-new-in-ruby-2-7-79c98b265502> for details on many of the new features.
+
+### Ohai 16 Improvements
+
+Ohai has been improved to gather additional system configuration information for use when authoring recipes and resources.
+
+#### Extended Azure Metadata
+
+The Azure Ohai plugin now gathers the latest version of the metadata provided by the Azure metadata endpoint. This greatly expands the information available on Azure instances. See [Ohai PR 1427](https://github.com/chef/ohai/pull/1427) for an example of the new data gathered.
+
+#### New Ohai Plugins
+
+New `ipc` and `interupts` plugins have been added to Ohai. The IPC plugin exposes SysV IPC shmem information and interutps plugin exposes data from `/proc/interrupts` and `/proc/irq`. Both of these plugins are disabled by default you you'll need to add :Ipc or :Interupts Thanks [@jsvana](https://github.com/jsvana) and [@davide125](https://github.com/davide125) for these new plugins.
+
+Note: Both of these plugins are optional plugins which are disabled by default. They can be enabled via your `client.rb`:
+
+```ruby
+ohai.optional_plugins = [
+  :IPC,
+  :Interups
+]
+```
+
+#### Improved Linux Network Plugin Data
+
+The Linux Network plugin has been improved to gather additional information from the `ethtool` utility. This includes the number of queues (`ethtool -l`), the coalesce parameters (`ethtool -c`), and information about the NIC driver (`ethtool -i)`. Thanks [@matt-c-clark](https://github.com/matt-c-clark) for these improvements.
+
+#### Windows DMI plugin
+
+Windows systems now include a new DMI plugin which presents data in a similar format to the DMI plugin on *nix systems. This makes it easier to detect system information like manufacturer, serial number or asset tag number in a cross platform way.
+
+## New Platforms
+
+Over the last quarter we've worked to greatly expand the platforms we support and produce packages for with the addition of Chef Infra Client packages for Ubuntu 20.04 amd64, Amazon Linux 2 x86_64/aarch64, and Debian 10 amd64. With the release of Chef Infra Client 16 we've expanded our platform support again with the following new platforms:
+
+- RHEL 8 aarch64
+- Ubuntu 20.04 aarch64
+- SLES 16 aarch64
+
+## Newly Introduced Deprecations
+
+Several legacy Windows helpers have been deprecated as they will always return true when running on Chef Infra Client's currently supported platforms. The helpers previously detected systems before Windows 2012 and systems running Windows Nano, which has been discontinued by Microsoft. These helpers were never documented externally so their usage is most likely minimal, but a new Cookstyle rule has been introduced to detect the usage of `older_than_win_2012_or_8?`: [ChefDeprecations/DeprecatedWindowsVersionCheck](https://github.com/chef/cookstyle/blob/master/docs/cops_chefdeprecations.md#chefdeprecationsdeprecatedwindowsversioncheck)
```

Comment:
```suggestion
Several legacy Windows helpers have been deprecated as they will always return true when running on Chef Infra Client's currently supported platforms. The helpers previously detected systems before Windows 2012 and systems running Windows Nano, which has been discontinued by Microsoft. These helpers were never documented externally, so their usage is most likely minimal. A new Cookstyle rule has been introduced to detect the usage of `older_than_win_2012_or_8?`: [ChefDeprecations/DeprecatedWindowsVersionCheck](https://github.com/chef/cookstyle/blob/master/docs/cops_chefdeprecations.md#chefdeprecationsdeprecatedwindowsversioncheck).
```

Reply:
I changed this to prior to

Reply:
I'm not sure what you mean by that. My suggested edit included creating a new sentence, which I don't see reflected so far.

Reply:
Now I'm following, but the other edits still stand.